source/ProcessManager.Abstractions/Api/Model/ScheduleOrchestrationInstanceCommand.cs

```diff
@@ -33,19 +32,53 @@ public abstract record ScheduleOrchestrationInstanceCommand<TInputParameterDto>
     /// <param name="orchestrationDescriptionUniqueName">Uniquely identifies the orchestration description from which the
     /// orchestration instance should be created.</param>
     /// <param name="runAt">The time when the orchestration instance should be executed by the Scheduler.</param>
-    /// <param name="inputParameter">Contains the Durable Functions orchestration input parameter value.</param>
     public ScheduleOrchestrationInstanceCommand(
         UserIdentityDto operatingIdentity,
         OrchestrationDescriptionUniqueNameDto orchestrationDescriptionUniqueName,
-        TInputParameterDto inputParameter,
         DateTimeOffset runAt)
-            : base(operatingIdentity, orchestrationDescriptionUniqueName, inputParameter)
+            : base(operatingIdentity)
```

Comment:
```suggestion
            : base(operatingIdentity, orchestrationDescriptionUniqueName, runAt)
```
Why do we only pass operationIdentity to the base?